frontend/appflowy_flutter/lib/plugins/database_view/grid/presentation/widgets/header/field_type_option_editor.dart

```diff
@@ -113,20 +112,12 @@ class _SwitchFieldButton extends StatelessWidget {
   Widget _buildMoreButton(BuildContext context) {
     final bloc = context.read<FieldTypeOptionEditBloc>();
     return FlowyButton(
-      hoverColor: AFThemeExtension.of(context).lightGreyHover,
       text: FlowyText.medium(
         bloc.state.field.fieldType.title(),
-        color: AFThemeExtension.of(context).textColor,
       ),
       margin: GridSize.typeOptionContentInsets,
-      leftIcon: svgWidget(
-        bloc.state.field.fieldType.iconName(),
-        color: Theme.of(context).iconTheme.color,
-      ),
-      rightIcon: svgWidget(
-        "grid/more",
-        color: Theme.of(context).iconTheme.color,
-      ),
+      leftIcon: FlowySvg(name: bloc.state.field.fieldType.iconName()),
```

Comment:
Is there a BlocBuilder somewhere higher in the widget tree? If the state changes, but no widgets in the hierarchy rebuild in response to state changes, this state here will be stale. 

Consider using a `BlocBuilder`/`BlocListener` here if this widget needs to be used somewhere where there is no `BlocBuilder` in the widget hierarchy.

Reply:
You can find the `BlocBuilder` on line 43 of this file.
![image](https://user-images.githubusercontent.com/14248245/235495651-ae74a2b3-2a91-43a8-ac62-29a3a4513fc2.png)
It built a `ListView` to display the editor options corresponding to the current property type(highlighted in green background). 
![image](https://user-images.githubusercontent.com/14248245/235496012-aa0610db-3998-451f-9f70-ae5d5a1dd26b.png)